### ISSUE TITLE
latex report errors

### ISSUE THREAD

**Issue by NicholasFry** [oemof/tespy#281]
This is a note. The latex report is missing a begin{document} and end{document} segment that will not allow for direct use in a compiler, that I know of. Perhaps that is an easy fix for the latex output, though that is far from my document style preference.

**Comment by NicholasFry** [comment]
Thanks for this @fwitte I will take another look at the documentation. I would not normally use the latex, but I am hoping to make a web application for a heat pump configuration in TESPy. Ideally, it should output the report and some matplotlib or seaborn plots to the user. If I get it deployed, I will be sure to cite your work.

**Comment by fwitte** [comment]
That sounds interesting, let me know if you need any assistance. I would be glad to help you. We could also link that as an example/tutorial in the documentation of the software. My initial idea of the report feature was to enable easy and transparent communication of model specifications and results for publications in scientific journals. However, in many cases e.g. markdown would be useful, too, I think. Or maybe some dash/plotly integration, which has very nice features especially for web applications (https://dash-gallery.plotly.host/Portal/). Running tespy from a dash web application should actually be quite easy, I think.

In any case, if you create something similar to the LaTeX report, maybe some parts or features of that might be interesting for other users as well and we can integrate the code into the software.

Best regards, have a nice day

**Comment by NicholasFry** [comment]
@fwitte I did get some better outputs by following your instruction. I do not currently know of a compiler to run on the backend before delivering simulation results to a user. This should be ok, however. I have a minimum viable product working for a single stage heat pump. It is not deployed yet as I do intend to add some of the plotly features, etc. This is what I came up with over the last few days: https://github.com/NicholasFry/Heat-pump-DH-web-application

Thanks again!
